src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py

```diff
@@ -101,6 +102,33 @@ def boost_low_volume_projects(context: TaskContext) -> None:
             boost_low_volume_projects_of_org.delay(org_id, projects_with_tx_count_and_rates)
 
 
+@instrumented_task(
+    name="sentry.dynamic_sampling.boost_low_volume_projects_of_org_with_query",
+    queue="dynamicsampling",
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=25 * 60,
```

Comment:
`soft_time_limit` should be shorter than the `time_limit`, `soft_time_limit` raises exception which can be caught so that the clean up (if necessary) could be done inside the task, while `time_limit` will just kill the task

Reply:
I see now that we also have this mistake `boost_low_volume_projects_of_org` 😬

Reply:
Yes, this was copied from there. Will investigate.

Reply:
Just sharing link to the docs with more info: https://docs.celeryq.dev/en/stable/userguide/workers.html#time-limits

Reply:
Additionally, that's a limit of 25 minutes, which is rather long. Let's lower it to something reasonable.